src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/add-polling-source-form.types.ts

```diff
@@ -26,6 +27,7 @@ export interface JsonFormControl {
     validators: JsonFormValidators;
     dataTestId?: string;
     list?: string[];
+    readFormatDescriptors?: Pick<RadioControlType, "label" | "value">[];
```

Comment:
I don't see any motivation for such a complex type. No relation between types - it should be a new interface with 2 fields.

Reply:
Changed to simple interface